apps/web/app/api/passkey/README.md

```
@@ -0,0 +1,132 @@
+# Usage
+
+## API Endpoints
+
+- **Generate Registration Options:**
+
+  `POST /api/passkey/generate-registration-options`
+
+  - Request Body: `{ "identifier": "user@example.com" }`
+  - Success Response (200):
+    ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)